### PR TITLE
[SM-1268] Review Java docs

### PR DESCRIPTION
**Pull request by mzieniukbw**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/SM-1268

## 📔 Objective

Review Java language docs.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: languages/java/README.md [hunk 1]
```diff
@@ -59,7 +61,7 @@ bitwardenClient.secrets().update(secretId, key2, value2, note2, organizationId,
 ### List secrets
 
 ```java
-var secretIdentifiersResponse secretIdentifiersResponse = bitwardenClient.secrets().list(organizationId);
+var secretIdentifiersResponse = bitwardenClient.secrets().list(organizationId);
 ```
 
 # Delete secret or project
```
Comment (Thomas-Avery) [inline languages/java/README.md:67]:
```suggestion
### Delete secret or project
```

**Review by Thomas-Avery — CHANGES_REQUESTED** [review]
Looks good!

File: languages/java/README.md [hunk 2]
```diff
@@ -50,24 +57,29 @@ var secretResponse = bitwardenClient.secrets().create(key, value, note, organiza
 UUID secretId = secretResponse.getID();
 ```
 
+### Get secret
+```java
+var secretResponse = bitwardenClient.secrets().get(secretId);
+```
```
Comment (Thomas-Avery) [inline languages/java/README.md:63]:
```suggestion
### Get secret

```java
var secretResponse = bitwardenClient.secrets().get(secretId);
```
```

**Review by Thomas-Avery — CHANGES_REQUESTED** [review]
Was able to follow the install guide and run this locally on macOS.

One formatting suggestion for consistency.

File: languages/java/README.md [hunk 3]
```diff
@@ -50,24 +57,29 @@ var secretResponse = bitwardenClient.secrets().create(key, value, note, organiza
 UUID secretId = secretResponse.getID();
 ```
 
+### Get secret
+
+```java
+var secretResponse = bitwardenClient.secrets().get(secretId);
+
```
Comment (Thomas-Avery) [inline languages/java/README.md:64]: Needs closing ``` 

I think the suggestion didn't read the closing right earlier.

Comment (mzieniukbw) [reply languages/java/README.md:65]: Nice spot, fixed!

**Review by Thomas-Avery — APPROVED** [review]
LGTM

**Comment by mzieniukbw** [comment]
@Thomas-Avery Do we need QA testing, since i have updated the Java build scripts ?
At the moment it is hard to even sanity check, since the Java build GH Action does not include the packaged Jar as artifact.

**Review by dani-garcia — APPROVED** [review]

**Pull request merged**